iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -32,399 +32,229 @@
 #include "ts/ink_defs.h"
 #include "ts/hugepages.h"
 
+#include <algorithm>
+
 /// Global singleton.
 class EventProcessor eventProcessor;
 
-class ThreadAffinityInitializer : public Continuation
```

Comment:
Is removing all this correct? This looks like the stuff that @SolidWallOfCode added recently.